.github/workflows/test.yml

```
@@ -1,4 +1,4 @@
-on: [push, pull_request]
+on: [push]
```

Would this turn off CI runs for PRs from forks? I think we may want to use something like the following:

```suggestion
on:
  push:
    branches:
      - main
  pull_request:
```
I believe this would cause the workflow to run on any opened PR/new commit to an open PR, and also after a merge to `main`.